x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/parser/ExpressionBuilder.java

```diff
@@ -185,6 +199,13 @@ public Expression visitOperatorExpressionDefault(EqlBaseParser.OperatorExpressio
         }
     }
 
+    private Expression combineExpressions(
+        List<? extends ParserRuleContext> expressions,
+        java.util.function.Function<Expression, Expression> mapper
+    ) {
```

Comment:
was the formatting intentional?

Reply:
No. It's whatever the IDE gives me and personally, I'm tired of fighting with it...
If anybody has suggestions, I'm all ears.

Reply:
Btw, I double check the length of the line and the break is intentional as in one line, it goes beyond the length. It looks unexpected but makes sense.

Reply:
Sure, sure. I was only referring to the `) {` on a new line, but irrelevant ultimately.